app/org/sagebionetworks/bridge/services/AuthenticationService.java

```diff
@@ -238,6 +254,40 @@ public void resetPassword(PasswordReset passwordReset) throws BridgeServiceExcep
         accountDao.resetPassword(passwordReset);
     }
     
+    private void repairConsents(Study study, Account account, UserSession session, ClientInfo clientInfo){
```

Comment:
Minor: Add some clarifying comments on why we need to do this, so future us don't look back on this code and get confused.

Some explanation about migrating from UserConsent2 to UserConsent3 and that wasn't completed successfully and how that left a bunch of accounts that have consent sigs in Stormpath but not in DDB, and that account.getActiveConsentSignature() gets consent sigs from Stormpath and how user.getConsentStatuses() gets consent sigs from DDB.